### ISSUE TITLE
Learning rate setting not available for Pytorch models

### ISSUE THREAD

**Issue by Antoine-Schwartz** [awslabs/gluon-ts#2288]
Hello everyone,

I hesitated to tag this issue as a "bug" because I think it's a simple design problem.

Indeed, unlike the **MXNet** equivalent, in the **Pytorch** estimators, the trainer does not carry the `learning_rate` information. 
It's defined in LightningModule, but this one is not instantiated before training: 

```
def train_model(
    self,
    training_data: Dataset,
    validation_data: Optional[Dataset] = None,
    num_workers: int = 0,
    shuffle_buffer_length: Optional[int] = None,
    cache_data: bool = False,
    ckpt_path: Optional[str] = None,
    **kwargs,
) -> TrainOutput:transformation = self.create_transformation()

    transformed_training_data = transformation.apply(
        training_data, is_train=True
    )
    
    training_network = self.create_lightning_module()  <== here
    
    ...
    
    trainer.fit(
            model=training_network,
            train_dataloaders=training_data_loader,
            val_dataloaders=validation_data_loader,
            ckpt_path=ckpt_path,
        )
    
    logger.info(f"Loading best model from {checkpoint.best_model_path}")
    best_model = training_network.load_from_checkpoint(
        checkpoint.best_model_path
    )
    
    return TrainOutput(
        transformation=transformation,
        trained_net=best_model,
        trainer=trainer,
        predictor=self.create_predictor(transformation, best_model),
    )
```

So I have the impression that it's not possible to set the learning rate at the estimator initialization, nor to specify it later during the `train` call.

Am I missing something?

Thanks in advance !

**Comment by melopeo** [comment]
Potentially related to PR https://github.com/awslabs/gluonts/pull/2287
